scaphandre/metadata.csv

```diff
@@ -0,0 +1,30 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+scaphandre.host.cpu.frequency,gauge,,,,,0,scaphandre,scaphandre host freq,
```

Comment:
This line is missing a description string.

Reply:
added